enginetest/load_queries.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package enginetest
+
+import (
+	"fmt"
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+const (
+	tableNameConst = "LOADTABLE"
+)
+
+var LoadDataScripts = []ScriptTest{
+	{
+		Name: "Basic load data with enclosed values.",
+		SetUpScript: []string{
+			fmt.Sprintf("create table %s(pk int primary key)", tableNameConst),
```

Comment:
really no reason to have this const. Just hardcode it everywhere it's needed. Makes the tests easier to read